files/eni-max-pods.txt

```diff
@@ -134,6 +134,7 @@ d3en.4xlarge 38
 d3en.6xlarge 58
 d3en.8xlarge 78
 d3en.xlarge 10
+dl1.24xlarge 2942
```

Comment:
@mikestef9 pointed out that this isn't right. With the VPC CNI, only the first network card is used and the rest are reserved for use by the EFA. 

So rather than `2942`, which was `60 * (50 - 1) + 2`, I'll change this value to `737` which is `15 * (50 - 1) + 2`. 

We'll need to also embed this logic within these two scripts - 

https://github.com/aws/amazon-vpc-cni-k8s/blob/master/scripts/gen_vpc_ip_limits.go#L146
https://github.com/awslabs/amazon-eks-ami/blob/master/files/max-pods-calculator.sh